pkg/subscriber/github/git_subscriber_item.go

```diff
@@ -323,29 +328,30 @@ func (ghsi *SubscriberItem) subscribeResourceFile(hostkey types.NamespacedName,
 	pkgMap[dpltosync.GetName()] = true
 
 	klog.V(4).Info("Ready to register template:", hostkey, dpltosync, syncsource)
+	err = ghsi.synchronizer.AddTemplates(syncsource, hostkey, []kubesynchronizer.DplUnit{{Dpl: dpltosync, Gvk: *validgvk}})
 
-	err = ghsi.synchronizer.RegisterTemplate(hostkey, dpltosync, syncsource)
 	if err != nil {
 		err = utils.SetInClusterPackageStatus(&(ghsi.Subscription.Status), dpltosync.GetName(), err, nil)
 
+		ghsi.successful = false
```

Comment:
minor: put this before the utils.SetInClusterPackageStatus

Reply:
I don't get it... why there's a difference? Please explain a bit more

Reply:
err = xyz()
if err

Should be together for better code readability. 

Also, logic flow wise its better. Setting it to false right away its quick than call the potential longer running function and wait then set.

Reply:
done